make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
There's already a makefile, where this may be implemented. Is there a reason to put a windows script here as this then will not be platform-independent. Draw an issue if you would like to change that, but not knowing how to accomplish that.